src/components/Header/Header.scss

```diff
@@ -15,15 +15,19 @@
   z-index: 1;
 
   .wkp-header-right {
-    margin-right: 10px;
+    height: 100%;
     display: flex;
-
-    .wkp-login {
-      margin-right: 45px;
-
-      a {
-        color: grey;
-      }
+    align-items: center;
+    justify-content: space-between;
+    position: absolute;
+    right: 10px;
```

Comment:
you should have to use a position: absolute here , display:flex should do the job.

Reply:
With this styling I ensure the header buttons are still well positioned, and the spacing between is correct in all the different languages.
Changing languages change the width of the buttons, and it is the only correct way I found to ensure the good display.

Reply:
ok for the `right` property  , but the ` left` is totally useless .